client/state/sharing/keyring/schema.js

```diff
@@ -0,0 +1,29 @@
+export const itemSchema = {
+	type: 'object',
```

Comment:
Could we maybe put `additionalProperties` at the top, so it's easier to match with patternProperties?